flow/core/kernel/lane_area_detectors/base.py

```diff
@@ -57,31 +57,42 @@ def update(self, reset):
     def get_ids(self):
         """Return the ids of all lane area detectors."""
         raise NotImplementedError
-
+    
+    """All following methods are using the API of the lane area detectors from traci:
+    https://sumo.dlr.de/docs/TraCI/Lane_Area_Detector_Value_Retrieval.html
+    """
```

Comment:
All methods are basically using the traci API for lane area detectors. I updated the docs, but can not give more detailed information right now.